### ISSUE TITLE
Need method id field and method table for citations

### ISSUE THREAD

**Issue by kjacobsen16** [USGS-WiM/GageStatsServices#8]
Each citation should have a method id field added, which would be associated with a method type from a new table. This is technically separate from the way we'll be doing it for the nss equations, but might be the same table, will figure out after we get the method options.

Waiting on Pete to send me the method options

**Comment by kjacobsen16** [comment]
@USGSPMM3 Pete - I was working on getting the method in NSS and realized I had created this separate issue for GageStats.. Was I misunderstanding when I created this issue or is there also a method used for gage stats?

This also mentions it being attached to the citation (for NSS you said it'd be regression region).

**Comment by kjacobsen16** [comment]
The matching NSSServices issue is [here](https://github.com/USGS-WiM/NSSServices/issues/115) (almost complete).

**Comment by USGSPMM3** [comment]
I don't believe we need to know these methods for the gage statistics, only for the regression equations. I think that is the question?

**Comment by kjacobsen16** [comment]
Alright, closing 👍
